webrtc/src/rtp_transceiver/rtp_sender/mod.rs

```diff
@@ -491,16 +621,10 @@ impl RTCRtpSender {
         lock.clone()
     }
 
-    pub(crate) fn set_initial_track_id(&self, id: String) -> Result<()> {
-        let mut lock = self.initial_track_id.lock().unwrap();
-
-        if lock.is_some() {
-            return Err(Error::ErrSenderInitialTrackIdAlreadySet);
-        }
-
-        *lock = Some(id);
+    pub(crate) fn associated_media_stream_ids(&self) -> Vec<String> {
```

Comment:
This order change between this and `set_initial_track_id` is just noise, please revert